app/controllers/admin/accounts_controller.rb

```diff
@@ -2,8 +2,11 @@
 
 module Admin
   class AccountsController < BaseController
+    helper_method :account_filter_params
+
     def index
-      @accounts = filtered_accounts.page(params[:page])
+      @account_filter = Form::AccountFilter.new(account_filter_params)
```

Comment:
well... I thought that the module included `ActiveModel::Model` should be name `Form::...` in this project.